tests/spec/core/lint-hrefs-ids-spec.js

```diff
@@ -0,0 +1,43 @@
+"use strict";
+describe("Core — Lint href's matching id's", function() {
+  afterAll(function(done) {
+    flushIframes();
+    done();
+  });
+  it("should warn about href pointing nowhere", function(done) {
+    var ops = {
+      config: makeBasicConfig(),
+      body:
+        makeDefaultBody() +
+          "<section>" +
+          "<a href='#test'>foo</a>" +
+          "</section>",
+    };
+    makeRSDoc(ops, function(doc) {
+      var ui = doc.defaultView.respecUI;
+      expect(ui.getErrors().length).toEqual(0);
```

Comment:
this is going to be very fragile... I'm ok to leave it for now, but will definitely need to come up with a better way to subscribe to these events.